deploy/openshift/collector/3-collector-deployment.yaml

```diff
@@ -24,5 +24,6 @@ spec:
         command:
         - /wavefront-collector
         - --source=kubernetes.summary_api:${MASTER_URL}?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250
+        - --source=prometheus:''?url=http://kube-state-metrics.kube-system.svc.cluster.local:8080/metrics
```

Comment:
Latest version of the collector now auto discovers the kube state service as it has the `prometheus.io/scrape` flag set to true. So this static configuration is not required. Adding this will result in the collector scraping the same target twice.

Reply:
See [discovery](https://github.com/wavefrontHQ/wavefront-kubernetes-collector/blob/master/docs/discovery.md) for details